rotkehlchen/exchanges/exchange.py

```diff
@@ -477,45 +478,49 @@ def query_history_with_callbacks(
             self,
             start_ts: Timestamp,
             end_ts: Timestamp,
-            success_callback: ExchangeHistorySuccessCallback,
             fail_callback: ExchangeHistoryFailCallback,
+            new_step_data: Optional[Tuple[ExchangeHistoryNewStepCallback, str]] = None,
     ) -> None:
         """Queries the historical event endpoints for this exchange and performs actions.
-
-        In case of success passes the result to success_callback.
+        The results are saved in the DB.
         In case of failure passes the error to failure_callback
+
+        `new_step_data` argument contains callback and exchange name to be used for steps.
         """
+        if new_step_data is not None:
+            new_step_callback, exchange_name = new_step_data
```

Comment:
```suggestion
        if new_step_data is not None:
            new_step_callback, exchange_name = new_step_data
            new_step_callback(f'Querying {exchange_name} trades history')
```

And delete it from below. No need to repeat the if check like this